lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -82,6 +87,14 @@ class MysqliStatement implements \IteratorAggregate, Statement
      */
     protected $_values = [];
 
+    /**
+     * Contains values from bindValue() that need to be sent
+     * using send_long_data *after* bind_param has been called.
+     *
+     * @var string[]|resource[]
+     */
+    protected $longData = [];
```

Comment:
Please declare it `private` instead. This data is not supposed to be reused via inheritance.

Reply:
done